### PR TITLE
Core profile currents

### PR DESCRIPTION
**Pull request by TimSlendebroek**
added core_profiles_currents_summary plot

added get_j_actuator_from_core_sources method (used in core_profiles_currents)

Simple but effective

image

### CODE REVIEW

**Comment by jmcclena** [comment]
@TimSlendebroek Why is this a separate pull request from https://github.com/gafusion/omas/pull/170 ?

**Comment by TimSlendebroek** [comment]
Good question, I forgot I pulled your changes to try NEO last Friday

File: omas/omas_plot.py [hunk 1]
```diff
@@ -1040,6 +1040,52 @@ def equilibrium_summary(ods, time_index=None, time=None, fig=None, ggd_points_tr
...
+        fig = pyplot.figure()
+
+    # time animation
+    time_index, time = handle_time(ods, 'core_profiles', time_index, time)
+    if isinstance(time_index, (list, numpy.ndarray)):
+        if len(time) == 1:
+            time_index = time_index[0]
+        else:
+            return ods_time_plot(
+                core_currents, ods, time_index, time, fig=fig, ax=axs, **kw
```
Comment (orso82) [inline omas/omas_plot.py:1075]:
```suggestion
                core_currents_summary, ods, time_index, time, fig=fig, ax=axs, **kw
```

**Comment by orso82** [comment]
@TimSlendebroek can you please add the plotting to be included in #170. Notice that in #170 we also changed the name of one of the routine. For extra kudos points, could you also please add a docstring to `core_sources_j_parallel_sum`? :)

**Comment by orso82** [comment]
@TimSlendebroek notices that to avoid conflicts with #170 it's probably simpler to simply close this PR and copy the `core_currents_summary()` plotting function to the `jact` branch.

**Pull request closed without merging**